File: data.tracker/ClientApp/src/models/entities/TimeTracking/TimeRegister.ts
import { EntityBase } from "../entity.base";
import { Project } from "../Projects/Project";

export class TimeRegister extends EntityBase {
  public Starts: string;
  public Ends: string;
  public Duration: number;
  public ProjectId: number;
  public Notes: string;
  public Project: Project;
  public ProjectName: string;
}

File: domain.tracker/Entitites/TimeTracking/TimeRegister.cs
using domain.tracker.Entitites.Projects;

namespace domain.tracker.Entitites.TimeTracking
{
    public sealed class TimeRegister : EntityBase
    {
        public DateTime Starts { get; set; }

        public DateTime Ends { get; set; }

        public int Duration { get; set; }

        public long ProjectId { get; set; }

        public string ProjectName { get; set; } = string.Empty;

        public string? Notes { get; set; }

        public Project Project { get; set; }
    }
}
